packages/package_info/example/android/app/src/androidTest/java/dev/flutter/plugins/packageinfoexample/MainActivityTest.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.plugins.packageinfoexample;
+
+import androidx.test.rule.ActivityTestRule;
+import dev.flutter.plugins.instrumentationadapter.FlutterRunner;
```

Comment:
[reminder from offline discussion]: update to the correct package name

Reply:
Done